04-single-hop-6lowpan-icmp/example_test_guide.md

```diff
@@ -0,0 +1,168 @@
+_The following is a possible way to run the tests, this is just an example_
+## Goal: Check single hop 6lowpan icmp
+
+_note: Since the ping6 function is now async some interval times may need to
+be increased to compensate_
+
+### Pre-setup Steps
+1. Get 2 of the following boards: `samr21-xpro`, `iotlab-m3`
+2. Get the following boards:`remote-revb`, `arduino-zero` with `xbee` module
+2. Go into the `cd ${RIOTBASE}/tests/gnrc_udp` directory
+2. Setup a terminal for each board _(tmux is your friend)_
+3. Find the serial numbers and ports with `ls /dev/serial/by-id/`
```

Comment:
Why not `make -C tests/gnrc_udp list-ttys` here?

Reply:
I think this is more standard...  Linux provides it.

Reply:
But... `list-ttys` is the RIOT way and also more portable.